src/Database/Redis/Cluster.hs

```diff
@@ -390,8 +390,9 @@ evaluateTransactionPipeline shardMapVar refreshShardmapAction conn requests' = d
     -- and let them decide how they would like to handle the error.
     when (any moved resps)
```

Comment:
should we refresh shard map when we got execabort?

Reply:
`EXECABORT` can be due to other reasons also, but if one of them is `MOVED` we can consider entire multi block commands are moved as we are doing this cross slot check before `hashSlot <- hashSlotForKeys (CrossSlotException requests) keys`. This is our error `[SingleLine "OK",Error "MOVED 13447 127.0.0.1:30003",Error "MOVED 13447 127.0.0.1:30003",Error "EXECABORT Transaction discarded because of previous errors."]`

Reply:
Ah okay. then